neo/Ledger/MemoryPool.cs

```diff
@@ -446,7 +446,7 @@ internal void UpdatePoolForBlockPersisted(Block block, Snapshot snapshot)
             _txRwLock.EnterWriteLock();
             try
             {
-                int blocksTillRebroadcast = unverifiedSortedTxPool == _sortedHighPrioTransactions
+                int blocksTillRebroadcast = Object.ReferenceEquals(unverifiedSortedTxPool,_sortedHighPrioTransactions)
```

Comment:
Please add a space after the comma so the code style matches.

Reply:
I had missed that one, thanks.